docs/tests/getting-started/github-actions.yml

```diff
@@ -11,10 +11,6 @@ env:
   # This needs to be unique across all of netlify.app
   APP_NAME: todoapp-dagger-europa
   NETLIFY_TEAM: dagger
-  # Get one from https://app.netlify.com/user/applications/personal
-  NETLIFY_TOKEN: ${{ secrets.NETLIFY_TOKEN }}
-  # GitHub Actions friendly logs
-  DAGGER_LOG_FORMAT: plain
```

Comment:
Not necessary with the dagger action as we append `--log-format plain` but I think we could use an env var instead of the flag in the action.

Reply:
If we append by default, we should remove, for sure.

Reply:
https://github.com/dagger/dagger-for-github/pull/50

Reply:
🚀